assets/js/blocks/cart-checkout/cart/index.js

```diff
@@ -51,4 +51,6 @@ const settings = {
 	},
 };
 
-registerBlockType( 'woocommerce/cart', settings );
+if ( WOOCOMMERCE_BLOCKS_PHASE > 1 ) {
+	registerBlockType( 'woocommerce/cart', settings );
```

Comment:
We could instead add a wrapper around `registerBlockType ` that supports a phase argument. It might make registration easier and improve DRY, and remove the need to import the settings everywhere.

Reply:
Love this! Maybe `registerGatedBlockType`?